README.rst

```diff
@@ -24,13 +24,28 @@ used docker containers
 Quick start
 ===========
 1. Set required variables in the ``env.config`` file
-2. Run ``./start_ci.sh up -d``
-3. Access ``http://PROXY_HOST`` in your browser:
+2. Ensure network doesn't overlap with already existing ones in your system.
```

Comment:
Ensure, that newly created network doesn't overlap with already existing ones in your system (see ``docker-compose.yml``)